### PR TITLE
Update cloudupload

### PR DESCRIPTION
**Pull request by Dulanic**
Added fix for https://github.com/madslundt/cloud-media-scripts/issues/54

image

### CODE REVIEW

File: scripts/cloudupload [hunk 1]
```diff
@@ -28,6 +28,14 @@ echo "[ $(date ${date_format}) ] ###### Start cloud upload ######"
 oldSize=0
 addedSize=0
 
+# Check if any files exist, if not exit
+file_ct=$(find "${local_decrypt_dir}" -type f | wc -l)
+
+if [ "$file_ct" = 0 ];then
+        echo "[ $(date ${date_format}) ] Nothing to upload, exiting"
+        exit 3
+else
```
Comment (madslundt) [inline scripts/cloudupload]: Shouldn't this be `fi`?

**Review by madslundt — CHANGES_REQUESTED** [review]
There is still a problem with the if else statement. It never ends.
A solution to this could be by replacing the else with fi

**Comment by Dulanic** [comment]
Thanks, I realized I put the fi at the end of my test script and thats why it worked. That smaller version is preferred. Change made.,

**Comment by madslundt** [comment]
Thank you! And great job by the way.

**Pull request merged**